contracts/CollectionPartyFactory.sol

```diff
@@ -60,43 +60,43 @@ contract PartyBuyFactory {
 
     function startParty(
         address _nftContract,
-        uint256 _tokenId,
         uint256 _maxPrice,
         uint256 _secondsToTimeout,
+        address[] calldata _deciders,
         Structs.AddressAndAmount calldata _split,
         Structs.AddressAndAmount calldata _tokenGate,
         string memory _name,
         string memory _symbol
-    ) external returns (address partyBuyProxy) {
+    ) external returns (address partyProxy) {
         bytes memory _initializationCalldata =
             abi.encodeWithSelector(
-            PartyBuy.initialize.selector,
+            CollectionParty.initialize.selector,
             _nftContract,
-            _tokenId,
             _maxPrice,
             _secondsToTimeout,
+            _deciders,
             _split,
             _tokenGate,
             _name,
             _symbol
         );
 
-        partyBuyProxy = address(
+        partyProxy = address(
             new NonReceivableInitializedProxy(
                 logic,
                 _initializationCalldata
             )
         );
 
-        deployedAt[partyBuyProxy] = block.number;
+        deployedAt[partyProxy] = block.number;
```

Comment:
can you remind me what we use this for/why we store it?

Reply:
it was a convenience so you could query the Factory to verify that a party was indeed deployed by the Factory (rather than needing to query logs) -- otherwise, a faux-Party could imitate the contract with malicious properties, and we didn't want the frontend to render that